.changeset/lucky-pianos-sing.md

```diff
@@ -0,0 +1,20 @@
+---
+'@openfn/language-common': major
+---
+
+### Changes
+
+- Added version locking to adaptors that uses `openfn/language-common`
+  dependency to ensure consistent and predictable behavior across different
+  environments.
+- Removed `http` on `common` to improve compatibility and reduce potential
+  security vulnerabilities.
+
+### Deprecated
+
+- Deprecated the use of _http.method()_ Eg: `http.post()` as it is no longer a
+  part of the `@openfn/language-common`.
+
+### Known Issues
```

Comment:
We don't need to include a known issues section if there are no known issues :) To be honest I would prefer not to release software if there are known issues in it...